### ISSUE TITLE
Question: which tool to open Computer Configuration/Policies/Administrative Templates/Windows Components/Internet Explorer/Internet Control Panel/Security?

### ISSUE THREAD

**Issue by handongw** [SlyngDK/play-module-ad-sso#2]
Hi, I am new to windows security stuff. I just installed my windows network with domain controller, dns, dhcp, windows 8.1, etc to try your project.

You mentioned to open Computer Configuration/Policies/Administrative Templates/Windows Components/Internet Explorer/Internet Control Panel/Security page,

which tool to use? how to navigate to the page?

many thx,

**Comment by handongw** [comment]
I managed to create and apply a group policy to enable Site to Zone Assignment List and add the sample play app to the list. I also created the keytab file and put it in conf directory of the sample as well as configure other application.conf properties. 

when I access the play app the first, it prompts me to type my windows user name and password, which, I did and can login successfully.

However, since I login to my domain in the windows 8.1 pc I expect I would login to the play web automatically w/o typing windows user name and password again. Any configurations did I missed?

thx a lot!
